week3/helpMenu.txt

```diff
@@ -0,0 +1,63 @@
+
+    HELP MENU                                                                                    ||
+=================                                                                                ||
+                                                                                                 ||
+Instructions:                                                                                    ||
+                                                                                                 ||
+---------------------------------------                                                          ||
+$ curl -X GET 'http://localhost:8080/' >>>>>>>>>>> To display the help menu                      ||
+---------------------------------------                                                          ||
+                                                                                                 ||
+---------------------------------------                                                          ||
+$ curl -X GET 'http://localhost:8080/help' >>>>>>>>>>> To display the help menu                  ||
+---------------------------------------                                                          ||
+                                                                                                 ||
+-----------------------------------------------------                                            ||
+$ curl -X POST 'http://localhost:8080/add/{newTask}' >>>>>>>>>>> To add a task to the list,      ||
+-----------------------------------------------------            where {newTask} without spaces. ||
+                                                                                                 ||
+-----------------------------------------------------                                            ||
+$ curl -X DELETE 'http://localhost:8080/delete/{taskId}' >>>>>>>>>>> To remove a task from the   ||
+-----------------------------------------------------             list, with a passed id         ||
+                                                                  {taskId}.                      ||
+--------------------------------------------                                                     ||
+$ curl -X PUT 'http://localhost:8080/reset' >>>>>>>>>>> To remove all the tasks.                 ||
```

Comment:
I would use DELETE instead of PUT for the reset endpoint, since it's a destructive operation.